### PR TITLE
Feature #303 - Migrate MatchMediaContext

### LINKED ISSUE #303: Migrate MatchMediaContext
Migrate MatchMediaContext provider to the wallet, it required to provide adaptive functionality to the `bierzo-wallet`.

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will migrate MatchMediaContext provider to `bierzo-wallet`. It will close #303.

### CODE REVIEW

File: packages/medulas-react-components/src/context/MatchMediaContext.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { throttle } from 'lodash';
+import * as React from 'react';
+
+const phoneScreen = 425;
+
+type MatchMediaProps = boolean;
+
+const isPhone = (): boolean => (window.matchMedia(`(min-width: ${phoneScreen}px)`).matches ? false : true);
+
+export const MatchMediaContext = React.createContext<MatchMediaProps>(false);
```
Comment (apanizo) [inline packages/medulas-react-components/src/context/MatchMediaContext.tsx]: This was an ugly hack I created for moving forward, now we should rely on hooks, check the [mui solution](https://material-ui.com/components/use-media-query/#using-material-uis-breakpoint-helpers).

I have stomachache each time I see my code 😓

Comment (apanizo) [reply packages/medulas-react-components/src/context/MatchMediaContext.tsx]: Also I would like to see an example of storybook where we use this hook.

I am expecting to see something:
```
const isPhone = useMediaQuery(...);
....
```

Comment (albertandrejev) [reply packages/medulas-react-components/src/context/MatchMediaContext.tsx]: What to best to use? `theme.breakpoints` or hardcoded value for the phone screen width like now?

Comment (kmw101) [reply packages/medulas-react-components/src/context/MatchMediaContext.tsx]: bierzo wallet is not designed for the phone, you could chop out the responsive views and adaptation for the phone?

**Comment by albertandrejev** [comment]
In order to check media change you can simple change orientation of viewport in the storybook shell.

File: packages/medulas-react-components/src/context/MatchMediaContext.tsx (hunk 1 above)
Comment (albertandrejev) [reply packages/medulas-react-components/src/context/MatchMediaContext.tsx]: ok

**Comment by albertandrejev** [comment]
According to @kmw101 comment (https://github.com/iov-one/ponferrada/pull/304#discussion_r292906663) this PR is not required anymore and can be closed.

**Comment by apanizo** [comment]
Good. Glad to see a code solution for using media queries in code. Good job.

**Pull request closed without merging**